src/config.cc

```diff
@@ -240,7 +260,8 @@ void Config::initFieldCallback() {
           if (cmd == "*") {
             profiling_sample_all_commands = true;
             profiling_sample_commands.clear();
-          } else if (srv->IsCommandExists(cmd)) {
+          } else if (Redis::IsCommandExists(cmd)) {
```

Comment:
So we must ask users to use origin name, kvrocks won't profile renamed command?

Reply:
redis_request.cc use isProfilingEnabled(original_cmd_name) (line 205), so profile_sample_commands need to use original cmd name